packages/teleport/src/Databases/AddDatabase/__snapshots__/AddDatabase.story.test.tsx.snap

```diff
@@ -1219,7 +1278,7 @@ exports[`render without token 1`] = `
                 $
               </div>
               <div>
-                teleport db start --token=[generated-join-token] --auth-server=localhost:443 --name=[db-name] --protocol=postgres --uri=[uri] --aws-region=[region]
+                teleport start --config=/etc/teleport.yaml
```

Comment:
`/etc/teleport.yaml` is the default location for the config file so this can probably be simplifies to just `teleport start`. Unless the intent is to explicitly demonstrate that you can set the config path via a flag.